### ISSUE TITLE
[IMPORTANT]False Render!?

### ISSUE THREAD

**Issue by hmnbh** [alexjlockwood/ShapeShifter#343]
Hello in stable version rendering working so perfect But in beta have the problem for following 
path data :
**"Dialpad vector from Android Studio"**

> **pathData** = `M 12 19 C 10.9 19 10 19.9 10 21 C 10 22.1 10.9 23 12 23 C 13.1 23 14 22.1 14 21 C 14 19.9 13.1 19 12 19 Z M 6 1 C 4.9 1 4 1.9 4 3 C 4 4.1 4.9 5 6 5 C 7.1 5 8 4.1 8 3 C 8 1.9 7.1 1 6 1 Z M 6 7 C 4.9 7 4 7.9 4 9 C 4 10.1 4.9 11 6 11 C 7.1 11 8 10.1 8 9 C 8 7.9 7.1 7 6 7 Z M 6 13 C 4.9 13 4 13.9 4 15 C 4 16.1 4.9 17 6 17 C 7.1 17 8 16.1 8 15 C 8 13.9 7.1 13 6 13 Z M 18 5 C 19.1 5 20 4.1 20 3 C 20 1.9 19.1 1 18 1 C 16.9 1 16 1.9 16 3 C 16 4.1 16.9 5 18 5 Z M 12 13 C 10.9 13 10 13.9 10 15 C 10 16.1 10.9 17 12 17 C 13.1 17 14 16.1 14 15 C 14 13.9 13.1 13 12 13 Z M 18 13 C 16.9 13 16 13.9 16 15 C 16 16.1 16.9 17 18 17 C 19.1 17 20 16.1 20 15 C 20 13.9 19.1 13 18 13 Z M 18 7 C 16.9 7 16 7.9 16 9 C 16 10.1 16.9 11 18 11 C 19.1 11 20 10.1 20 9 C 20 7.9 19.1 7 18 7 Z M 12 7 C 10.9 7 10 7.9 10 9 C 10 10.1 10.9 11 12 11 C 13.1 11 14 10.1 14 9 C 14 7.9 13.1 7 12 7 Z M 12 1 C 10.9 1 10 1.9 10 3 C 10 4.1 10.9 5 12 5 C 13.1 5 14 4.1 14 3 C 14 1.9 13.1 1 12 1 Z`

**Stable version**

![zzzzzzzzzz](https://user-images.githubusercontent.com/53038586/80028126-eb711000-84f9-11ea-8540-2fdb04ab312d.PNG)

**Beta version**

![zzzzzzzzzzz](https://user-images.githubusercontent.com/53038586/80028207-02176700-84fa-11ea-824d-55b25be63f58.PNG)

**Comment by alexjlockwood** [comment]
Yeah, it's a known bug in the beta version right now. It doesn't currently render `pathData` strings that contain multiple subpaths (i.e. strings that contain multiple `M` commands).

You would have to workaround by creating a separate path layer for each one, i.e.

- M 12 19 C 10.9 19 10 19.9 10 21 C 10 22.1 10.9 23 12 23 C 13.1 23 14 22.1 14 21 C 14 19.9 13.1 19 12 19 Z 
- M 6 1 C 4.9 1 4 1.9 4 3 C 4 4.1 4.9 5 6 5 C 7.1 5 8 4.1 8 3 C 8 1.9 7.1 1 6 1 Z M 6 7 C 4.9 7 4 7.9 4 9 C 4 10.1 4.9 11 6 11 C 7.1 11 8 10.1 8 9 C 8 7.9 7.1 7 6 7 Z 
- M 6 13 C 4.9 13 4 13.9 4 15 C 4 16.1 4.9 17 6 17 C 7.1 17 8 16.1 8 15 C 8 13.9 7.1 13 6 13 Z M 18 5 C 19.1 5 20 4.1 20 3 C 20 1.9 19.1 1 18 1 C 16.9 1 16 1.9 16 3 C 16 4.1 16.9 5 18 5 Z

etc.
